include/eve/module/core/regular/impl/simd/arm/sve/any.hpp

```diff
@@ -9,14 +9,17 @@
 
 #include <eve/detail/has_abi.hpp>
 #include <eve/detail/implementation.hpp>
+#include <eve/detail/remove_garbage.hpp>
 
 namespace eve::detail
 {
 template<scalar_value T, typename N, relative_conditional_expr C>
 EVE_FORCEINLINE bool
-any_(EVE_SUPPORTS(sve_), C const& cond, logical<wide<T,N>> const& v) noexcept
+any_(EVE_SUPPORTS(sve_), C const& cond, logical<wide<T,N>> v) noexcept
 requires sve_abi<abi_t<T, N>>
 {
+  v = remove_garbage(v);
```

Comment:
I don't like this very much.
Like for `svptest_any` we can put this remove garbage mask directly there.

Reply:
Updated